Readme.md

```diff
@@ -10,91 +10,15 @@ Do you want to quickly try out this project and see it in action? Check it out i
 
 ## Documentation
 
-### Async Editor
+Check out the [`Lombiq.ContentEditors.Samples` project](Lombiq.ContentEditors.Samples) for various examples in the form of an Orchard Core module.
 
-The Async Editor feature provides an infrastructure for creating editors that can load and save content asynchronously. The editor can optionally render multiple pages where each page can load and save data independently.
-
-#### Creating an Async Editor for content items
-
-To create an content type-specific async editor you need to create a provider class that inherits from `ContentItemAsyncEditorProviderBase<TProvider>` where `TProvider` is the name of your provider class. Example:
-
-    ```csharp
-    public class EmployeeAsyncEditorProvider : ContentItemAsyncEditorProviderBase<EmployeeAsyncEditorProvider>
-    {
-        public EmployeeAsyncEditorProvider(IContentItemAsyncEditorProviderServices<EmployeeAsyncEditorProvider> providerServices)
-        : base(providerServices)
-        {
-        }
-
-        protected override bool CanHandleContentType(string contentType) => contentType == ContentTypes.Employee;
-
-        public override Task<IEnumerable<AsyncEditorGroupDescriptor<ContentItem>>> DescribeEditorGroupsAsync() =>
-            Task.FromResult(new[]
-            {
-                DescribeEditorGroup(PersonalDetails, T["Personal Details"]),
-                DescribeEditorGroup(EmploymentDetails, T["Employment Details"], isPublishGroup: true),
-            }.AsEnumerable());
-    }
-    ```
-
-The `CanHandleContentType` method is used to determine if the provider can handle the content type.
-
-The `DescribeEditorGroupsAsync` method is used to describe the available editor groups. The above example describes two groups: `PersonalDetails` and `EmploymentDetails`. A group can be marked as the publish group meaning the user can publish the content item only on that editor page. The `DescribeEditorGroup` has two other parameters: `isAccessibleFactory` and `isFilledFactory`. These can be used to determine if the editor group is accessible (e.g., the user has permission to edit that group) and if the group is filled with data (e.g., the fill state is tracked somehow).
-
-Finally, make sure each of your shapes are placed in the proper editor group. You can use file _placement.json_ file for this. Example:
+Note: The `Lombiq.ContentEditors.sln` file exists to provide centralized building of the independent projects but otherwise you don't need to use it.
```

Comment:
```suggestion
Note: The _Lombiq.ContentEditors.sln_ file exists to provide centralized building of the independent projects but otherwise you don't need to use it.
```